lib/ansible/modules/cloud/digital_ocean/digital_ocean_account_facts.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_account_facts
+short_description: Gather facts about DigitalOcean User account
+description:
+    - This module can be used to gather facts about User account.
+author: "Abhijeet Kasurde (@akasurde)"
+version_added: "2.6"
+
+requirements:
+  - "python >= 2.6"
+
+extends_documentation_fragment: digital_ocean.documentation
+'''
+
+
+EXAMPLES = '''
+- name: Gather facts about user account
+  digital_ocean_account_facts:
+    oauth_token: "{{ oauth_token }}"
+'''
+
+
+RETURN = '''
+data:
+    description: DigitalOcean account facts
+    returned: success
+    type: dictionary
+    sample: {
```

Comment:
Change sample to `data` that will represent what is being returned to the user. 

Example task:
```yams
- debug:
    var: resp.data
```

Outputs:
```
    "resp.data": {
        "droplet_limit": 100,
        "email": "sammy@digitalocean.com",
        "email_verified": true,
        "floating_ip_limit": 20,
        "status": "active",
        "status_message": "",
        "uuid": "b0ea118ed29548573729203027a0442f9"
    }
```

Reply:
`sample` is actual table header in documentation - e.g., [DO Tag](http://docs.ansible.com/ansible/latest/digital_ocean_tag_module.html#return-values) return values table.